bloggies_frontend/src/RoutedPages/ComposePage/index.tsx

```diff
@@ -22,7 +22,8 @@ function ComposePage() {
   }
 
   const postBlog = async (data: PostFormData) => {
-    dispatch(addPostToAPI(data));
+     dispatch(addPostToAPI(data));
+     dispatch(getUserInfoFromAPI());
```

Comment:
i know it wasn't part of your ticket, but would you like to add an additional check to ensure that the user's `membership_status` is `ACTIVE` when users try to go to the `ComposePage`? If `not ACTIVE`, display an error that says something like `Only premium members can contribute posts.`

lmk what you think!